spring-boot-data-dynamodb/src/main/resources/logback.xml

```diff
@@ -0,0 +1,14 @@
+<configuration>
+
+	<appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+		<encoder>
+			<pattern>web - %date [%thread] %-5level %logger{36} - %message%n
+			</pattern>
+		</encoder>
+	</appender>
+
+	<root level="INFO">
+		<appender-ref ref="STDOUT" />
+	</root>
+
+</configuration>
```

Comment:
![image](https://cloud.githubusercontent.com/assets/4750426/19377388/d4355898-919a-11e6-9475-4d329ee28755.png)

It is 4 spaces.  The github UI makes it look like a tab.  @pkhanal